examples/lightgbm/conda.yaml

```diff
@@ -0,0 +1,9 @@
+name: lightgbm-example
+channels:
+  - defaults
+  - conda-forge
+dependencies:
+  - python=3.6
+  - lightgbm
+  - pip:
+      - git+https://github.com/mlflow/mlflow
```

Comment:
The current latest version (1.4.0) does not yet support the XGBoost and LightGBM flavors (the next version will). Should I wait for the next release?

Reply:
Good question, yeah we plan to release 1.5 some time next week, so maybe we should update this to mlflow==1.5 then.

Reply:
ok, I'll fix this.